REFACTORING_PLAN.md

```
@@ -805,12 +805,160 @@ _To be filled by completing agent_
+- `/hooks` (20 hooks)
+- `/lib/hooks` (7 hooks)
+- `/components/hooks` (1 hook)
+- `/components/dashboard/hooks` (1 hook)
+- `/components/organisms/hooks` (2 hooks)
+- `/components/admin/` (2 hooks inline)
+
+This makes discovery difficult and leads to potential duplication.
+
+**Solution:**
+Consolidate all hooks into `/hooks` with subdirectories by domain:
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)